bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -15,6 +15,8 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{{_('Welcome to Firefox Developer Edition')}}{% endblock %}
 {% block page_title_suffix %}{% endblock %}
+{% block page_image %}{{ static('img/firefox/developer/page-image.png') }}{% endblock %}
```

Comment:
Since this is a `/firstrun` page we should probably override `{% block page_og_url %}` too.